packages/terra-action-footer/src/clinical-lowlight-theme/BlockActionFooter.module.scss

```diff
@@ -0,0 +1,6 @@
+:local {
+  .clinical-lowlight-theme {
```

Comment:
Did we want to back-fill all missing css vars with these changes?

Reply:
Updated here https://github.com/cerner/terra-core/pull/2955/commits/6146fb9e5371674f76085d02d1f01acd52f3d3b4